src/main/java/com/example/onlinebookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.example.onlinebookstore.service.impl;
+
+import com.example.onlinebookstore.model.Book;
+import com.example.onlinebookstore.repository.BookRepository;
+import com.example.onlinebookstore.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    @Autowired
+    private BookRepository bookRepository;
```

Comment:
injection through a constructor is always a preferred way because with it we can guarantee that our bean will always be initialized